site/content/installation/ngf-images/pulling-ngf-image.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Getting the F5 Registry NGINX Plus Gateway Fabric Image
```

Comment:
```suggestion
title: Push NGINX Plus Image to Private Registry
```

maybe doc-ops can help phrase this better for a page title. Also, NGINX Plus Gateway Fabric is not the product name, so I think we should probably just say NGINX Plus image throughout the docs.

Reply:
updating to your suggestion for now